recipes/aws-c-common/all/test_package/CMakeLists.txt

```diff
@@ -3,5 +3,7 @@ project(test_package LANGUAGES C)
 
 find_package(aws-c-common REQUIRED CONFIG)
 
+include(AwsCFlags)
```

Comment:
TO check that the builddirs are correctly set

Reply:
Note that as we need the cmake files, the hook will warn that they are still present for this package, but this is expected